dataland-frontend/src/components/forms/parts/fields/FormFieldProps.ts

```diff
@@ -26,35 +26,73 @@ export const FormFieldProps = {
     type: Boolean,
     default: false,
   },
+  innerClass: {
+    type: String,
+    default: "short",
+  },
 };
 
-export const YesNoFormFieldProps = Object.assign(deepCopyObject(FormFieldProps), {
+export const YesNoFormFieldProps = Object.assign(deepCopyObject(BaseFormFieldProps), {
   certificateRequiredIfYes: {
     type: Boolean,
     default: false,
   },
 }) as Readonly<ComponentPropsOptions>;
-export const FormFieldPropsWithPlaceholder = Object.assign(deepCopyObject(FormFieldProps), {
+
+export const FormFieldPropsWithPlaceholder = Object.assign(deepCopyObject(BaseFormFieldProps), {
   placeholder: {
     type: String,
     default: "",
   },
 }) as Readonly<ComponentPropsOptions>;
+
 export const DateFormFieldProps = Object.assign(deepCopyObject(FormFieldPropsWithPlaceholder as ObjectType), {
   todayAsMax: {
     type: Boolean,
     default: false,
   },
 }) as Readonly<ComponentPropsOptions>;
+
 export const DropdownOptionFormFieldProps = Object.assign(deepCopyObject(FormFieldPropsWithPlaceholder as ObjectType), {
   options: {
     type: Array as () => Array<DropdownOption> | undefined,
     required: true,
   },
 }) as Readonly<ComponentPropsOptions>;
+
 export const OptionsFormFieldProps = Object.assign(deepCopyObject(FormFieldPropsWithPlaceholder as ObjectType), {
   options: {
-    type: Array as () => Array<typeof Option> | undefined,
+    type: Object,
+    required: true,
+  },
+}) as Readonly<ComponentPropsOptions>;
+
+export const SingleSelectFormElementProps = Object.assign(deepCopyObject(FormFieldPropsWithPlaceholder as ObjectType), {
+  options: {
+    type: Array as () => Array<DropdownOption>,
+    required: true,
+  },
+}) as Readonly<ComponentPropsOptions>;
+
```

Comment:
I just noticed that `DropdownOptionFormFieldProps` already covers this exact object.
So we can delete `SingleSelectFormElementProps` again and just use `DropdownOptionFormFieldProps` for the SingleSelect component.

Reply:
👍